source/administration/config-settings.rst

```diff
@@ -4577,6 +4577,24 @@ By default, in order to avoid leaking sensitive information, no method parameter
 | This feature's ``config.json`` setting is ``"EnableOpenTracing": false`` with options ``true`` and ``false``.                                                        |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Import Settings Default Directory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The directory where the imported files are stored. The path is relative to the ``FileSettings`` directory. By default, imports are stored under ``./data/import``.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``ImportSettingsDefaultDirectory: ./import`` with string input.                                                            |
```

Comment:
Technically the setting is just `Directory` which goes under the `ImportSettings` section. I am saying this because searching for the string `ImportSettingsDefaultDirectory` won't find any result.

This is how it would look in a `config.json` file:

```json
"ImportSettings": {
  "Directory": "./import",
  "RetentionDays": 30
}
```

Reply:
Thank you so much, @streamer45! I'll ensure this and the Telemetry documentation is updated accordingly.